src/components/search.js

```diff
@@ -1,20 +1,35 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import SVGs from '../svgs'
 import NimbleEmojiIndex from '../utils/emoji-index/nimble-emoji-index'
 
 export default class Search extends React.PureComponent {
   constructor(props) {
     super(props)
+    this.state = {
+      icon: SVGs.searching['search'],
+      tabbable: '',
```

Comment:
Can we name this something more descriptive? `canTabIcon` perhaps? Or `isIconTabbable`? Maybe even `isSearching`?
Also let's make it a `boolean`, not a `String`, and handle converting the boolean to the appropriate `tabIndex` in the `render` function.

```diff
diff --git a/src/components/search.js b/src/components/search.js
index 1813722..57b2a75 100644
--- a/src/components/search.js
+++ b/src/components/search.js
@@ -9,7 +9,7 @@ export default class Search extends React.PureComponent {
     super(props)
     this.state = {
       icon: SVGs.searching['search'],
-      tabbable: '',
+      isSearching: false,
     }

     this.data = props.data
@@ -23,12 +23,12 @@ export default class Search extends React.PureComponent {
     if (value == '')
       this.setState({
         icon: SVGs.searching['search'],
-        tabbable: '',
+        isSearching: false,
       })
     else
       this.setState({
         icon: SVGs.searching['delete'],
-        tabbable: '0',
+        isSearching: true,
       })

     this.props.onSearch(
@@ -58,7 +58,7 @@ export default class Search extends React.PureComponent {

   render() {
     var { i18n, autoFocus } = this.props
-    var { icon, tabbable } = this.state
+    var { icon, isSearching } = this.state

     return (
       <div className="emoji-mart-search">
@@ -72,7 +72,7 @@ export default class Search extends React.PureComponent {
         <div
           className="emoji-mart-search-icon"
           onClick={this.handleClick}
-          tabIndex={tabbable}
+          tabIndex={isSearching ? '0' : ''}
           onKeyDown={this.handleClick}
           dangerouslySetInnerHTML={{ __html: icon }}
         />
```